### PR TITLE
docs: move sso how to guides to the how-to folder

### PR DESCRIPTION
**Pull request by sighphyre**
We're in the process of moving how-to guides into the how-to folder, where they belong. This continues that work and shifts the how to guides for SSO to the correct place and gives them appropriate names

### CODE REVIEW

File: website/docs/how-to/how-to-add-sso-google.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 ---
-id: sso-google
 title: '[Deprecated] How to add SSO with Google'
 ---
 
-> Single Sign-on via the Google Authenticator provider is deprecated. We recommend using [OpenID Connect](./sso-open-id-connect.md) instead.
+> Single Sign-on via the Google Authenticator provider is deprecated. We recommend using [OpenID Connect](./how-to-add-sso-open-id-connect.md) instead.
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-add-sso-google.md:5]:
```suggestion
:::caution Deprecation notice

Single Sign-on via the Google Authenticator provider is deprecated. We recommend using [OpenID Connect](./how-to-add-sso-open-id-connect.md) instead.

:::
```

File: website/docs/how-to/how-to-add-sso-saml-keycloak.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
 ---
-id: sso-saml-keycloak
 title: How to add SSO with SAML 2.0 Keycloak
 ---
 
 > The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-add-sso-saml-keycloak.md:5]:
```suggestion
:::note Availability

The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.

:::
```

File: website/docs/how-to/how-to-add-sso-open-id-connect.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
 ---
-id: sso-open-id-connect
 title: How to add SSO with OpenID Connect
 ---
 
 > The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-add-sso-open-id-connect.md:5]:
```suggestion
:::note Availability

The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.

:::
```

File: website/docs/how-to/how-to-add-sso-saml.md [hunk 4]
```diff
@@ -1,13 +1,12 @@
 ---
-id: sso-saml
 title: How to add SSO with SAML 2.0 Okta
 ---
 
 > The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.
```
Comment (thomasheartman) [inline website/docs/how-to/how-to-add-sso-saml.md:5]:
```suggestion
:::note Availability

The **Single-Sign-On capability** is only available for customers on the Enterprise subscription. Check out the [Unleash plans](https://www.getunleash.io/plans) for details.

:::
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
You're doing the proverbial Lord's work here, Simon! 🙌🏼  This looks great, but changing the file locations also changes their URL, so you'll need to add the corresponding redirects:

Check out the `plugin-client-redirects` in `website/docusaurus.config.js`. Just add the old path and the new ones. Be aware that the redirects aren't active in development mode, so you'll need to use a production build to test that they work. There's more info in [docusaurus'  plugin-client-redirect docs](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects/).

Otherwise, I've also added suggestions to use docusaurus' admonitions for availability and deprecation notices. It's not really part of the relocation, but it might be nice to do regardless.

**Comment by sighphyre** [comment]
> You're doing the proverbial Lord's work here, Simon!

Gosh that made me blush. I think it's time, yeah, if we're going to go forwards with new patterns, let's go forwards. Happy to help

>Check out the plugin-client-redirects in website/docusaurus.config.js. Just add the old path and the new ones. Be aware that the redirects aren't active in development mode, so you'll need to use a production build to test that they work. There's more info in [docusaurus' plugin-client-redirect docs](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects/).

> Otherwise, I've also added suggestions to use docusaurus' admonitions for availability and deprecation notices. It's not really part of the relocation, but it might be nice to do regardless.

Amazing, thank you! I will take a look today, I figured there was probably something I was missing here so thanks for the pointer!

**Review by thomasheartman — APPROVED** [review]
This looks good to me now 😄

**Comment by thomasheartman** [comment]
I've also gone through and tested all the redirects you've added, by the way. They work as expected 🥳

**Comment by sighphyre** [comment]
> I've also gone through and tested all the redirects you've added, by the way. They work as expected partying_face

You're amazing, thank you!

**Pull request merged**